msi/release.sh

```diff
@@ -1,17 +1,56 @@
 #!/bin/sh
 
-# Build Git for Windows' .msi files
+me=$(basename $0)
+function showUsage() {
```

Comment:
Ah, I see what happened. I changed all the functions in `render-release-notes.sh` according to this comment, but forgot to do it in this file here. Sorry about that!